packages/react-components/react-tooltip/src/components/Tooltip/useTooltip.tsx

```diff
@@ -86,7 +86,7 @@ export const useTooltip_unstable = (props: TooltipProps): TooltipState => {
     arrowPadding: 2 * tooltipBorderRadius,
     position: 'above' as const,
     align: 'center' as const,
-    offset: [0, 4] as [number, number],
+    offset: { crossAxis: 0, mainAxis: 4 },
```

Comment:
(nit) here as well, does the shorthand work?
```suggestion
    offset: 4,
```